src/app/modules/item/components/item-children-edit/item-children.ts

```diff
@@ -1,6 +1,8 @@
+import { ItemType } from '../../../../shared/helpers/item-type';
 
 export interface ItemChild {
```

Comment:
If ItemChild is only used in this component, it could go (back) in the component maybe.

Reply:
Done